dashboard/README.md

```diff
@@ -196,23 +214,38 @@ The whole idea is to be able to reuse some 'widgets' when designing HTML pages
 
 So instead that each page make the design/css for all buttons, inputs, it should be defined in some widget components.
 
-The components are located in the src/components/widget folder
+The components are located in the `src/components/widget` folder
 
-It includes toggle buttons, selecter, etc.
+It includes toggle buttons, selector, etc.
 
 A demo page is also provided to browse them: localhost:5000/#/demo-components
 
-
 ## API of Che
 
 Each call to the Che API shouldn't be made directly from the controller of the page.
-For that, it has to use Che API fatories which are handling the job (with promises operations)
+For that, it has to use Che API factories which are handling the job (with promises operations)
 
 By injecting 'cheAPI' inside a controller, all operations can be called.
 
-for example cheAPI.getWorkspace().getWorkspaces() for getting the array of the current workspaces of the user
+for example `cheAPI.getWorkspace().getWorkspaces()` for getting the array of the current workspaces of the user
 
 Mocks are also provided for the Che API, allowing to emulate a real backend for unit tests or e2e tests
 
+## Configurability
 
+The `configuration.menu.disabled` field in [product.json](/src/assets/branding/product.json) allows users to list there menu entries they want to hide in left navigation bar. Along with that corresponding routes also will be disabled.
```

Comment:
It may be useful to have here a list of all available values that can be disabled. From source code I see it can be:
```
'administration' | 'factories' | 'getstarted' | 'organizations' | 'stacks'
```

Reply:
done